docs/kb/KHV028.md

```diff
@@ -0,0 +1,19 @@
+---
+id: KHV028
+title: Kubectl Vulnerable To CVE-2019-1002101
+categories: [Remote Code Execution]
+---
+
+# KHV028 - Kubectl Vulnerable To CVE-2019-1002101
+
+## Issue description
+
+Kubectl is found to be vulnerable to CVE-2019-1002101. Please see the Vulnerability description for additional information.
+
+## Remediation
+
+Please see the Vulnerability description for remediation.
+
+## References
+
+- [CVE-2019-11246: Another kubectl Path Traversal Vulnerability Disclosed](https://blog.aquasec.com/kubernetes-security-kubectl-cve-2019-11246)
```

Comment:
copy-and-paste? Wrong CVE ref

Reply:
actually no - the blog post linked is indeed about 11246 but it's also describing 1002101, that's why it's here. The link description is just the title of the page. What do you think a better description would be?

Reply:
Ah, I see. Maybe take the CVE reference out of the title? 

Could also add a reference to https://nvd.nist.gov/vuln/detail/CVE-2019-1002101

Reply:
done